test/core/fixtures/transformation/es6.spec.template-literals/only/expected.js

```diff
@@ -1,3 +1,3 @@
 "use strict";
 
-var foo = "" + String(test);
+var foo = String(test);
```

Comment:
If the original is `${test}`, I believe the spec defines that as a cast to string operation.